cli/tauri.js/src/helpers/logger.ts

```diff
@@ -0,0 +1,23 @@
+import chalk from 'chalk'
+import ms from 'ms'
+
+let prevTime: number
+
+export default (banner: string, color: string = 'green') => {
+  return (msg?: string) => {
+    const curr = +new Date()
+    const diff = curr - (prevTime || curr)
+
+    prevTime = curr
+
+    if (msg) {
+      console.log(
+        // TODO: proper typings for color and banner
+        // @ts-ignore
+        ` ${chalk[color](banner)} ${msg} ${chalk.green(`+${ms(diff)}`)}`
+      )
+    } else {
+      console.log()
```

Comment:
Would it make sense to use `process.stdout.write()` instead of `console.log()`? Its something I've been wondering about for a while. It would allow us to enable the linting rule for noconsole which I particularly prefer.

Reply:
Just seen this SO question: https://stackoverflow.com/questions/4976466/difference-between-process-stdout-write-and-console-log-in-node-js
But I dunno what would linters think about it instead.

Reply:
> It would allow us to enable the linting rule for noconsole which I particularly prefer.

If you want to disable the lint rule for a specific area you can do:

```js
/*eslint-disable-next-line no-console*/
console.log('hi')